### ISSUE TITLE
Concerns about Histogram Equalization

### ISSUE THREAD

**Issue by lanpokn** [iSEE-Laboratory/DiffUIR#19]
Greetings! This is an excellent work. However, I noticed an issue while using it. 

The processing for low-light enhancement is different from other tasks, as it requires histogram equalization during reading, while other tasks do not. This seems to go against the original intention of a "blind all-in-one" approach, where we should not know the type of the input image, and therefore should not decide whether to perform histogram equalization.

May I ask if there would be any issues **if we remove the histogram equalization operation** for the low-light enhancement task, and train and test using a unified approach for all tasks? Or, based on testing, **is histogram equalization necessary** for low-light enhancement?

**Comment by zhengdian1** [comment]
Yes, you are right. We really wanted to remove this histogram before submission, but due to time constraints, we couldn't adjust it, which is our regret. However, we believe that improving the noise schedule can solve this problem, and we also plan to expand it into a journal article and address this issue later.
